bitnami/rabbitmq/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: rabbitmq
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq
-version: 12.3.0
+version: 12.3.1
```

As this is a new feature, it should be a minor bump :) 
```suggestion
version: 12.4.0
```